multicluster/apis/multicluster/v1alpha1/tunnelendpoint_types.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2021 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TunnelEndpointSpec defines the desired state of TunnelEndpoint
+type TunnelEndpointSpec struct {
+	// Role (leader/member) shows the role of Gateway node if HA is enabled
```

Comment:
It should active/standby if it is for HA? And maybe a boolean of "Active".

Reply:
Do we plan to implement HA right now? If not, maybe design and add code later. For example, we might do active/active too.

Reply:
ok, we don't plan to support HA in first release, let me remove it. I will add a new design doc in google doc.